server.js

```diff
@@ -1,10 +1,23 @@
 const express = require('express')
 const app = express()
+const mongoose = require('mongoose')
+const URI = 'mongodb+srv://Rishabh:qwertyuiop@cluster0.8yyadat.mongodb.net/?retryWrites=true&w=majority'
+mongoose.connect(URI)
+.then(()=>{
+  app.get('/',(req,res)=>{
+    res.json({Connection_Status:'Connected'})
+  })
+}).catch((err)=>{
+  app.get('/',(req,res)=>{
+    res.json({Connection_Status:'Not Conncected'})
```

Comment:
It's a good practice to handle errors in a more informative way. Instead of just sending 'Not Connected', consider sending the actual error message to help with debugging. [medium]